manifest.yml

```diff
@@ -0,0 +1,16 @@
+name : "Device-Shadow-for-AWS-IoT-embedded-sdk"
+version: "1.0.0"
+description: |
+  "Client library for using AWS IoT Shadow service on embedded devices. \n"
+dependencies:
+ - name: "coreJSON"
```

Comment:
JSON is not a dependency for Shadows libs. We designed lib to be independent of JSON and MQTT.

Reply:
Good point. It should be dependency on Shadow demo in hub ( when we decide to add it) .